IPython/core/release.py

```diff
@@ -121,5 +119,8 @@
     'Programming Language :: Python :: 2',
     'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
```

Comment:
I think we've discussed this before, but I don't think we should use the specific Python 3.x classifiers. There's no particular reason it wouldn't be compatible with 3.6, or later versions, so this can never be complete. And as far as I know, no tools actually look at the more specific classifiers anyway. I think we should just use the more general `Python :: 3` classifier, and describe in prose that 3.3 or above is required (i.e. as we already do).